### PR TITLE
fix plot size in `png` - rework docs

### PR DESCRIPTION
**Pull request by t-bltg**
- fix incorrect plot size when exporting to png (buggy `polarplot`);
- cache font for speeding up `png_image` & `savefig`;
- add banner for pending `Term` 1.0.

### CODE REVIEW

**Comment by t-bltg** [comment]
We need to cache the font since we hit https://github.com/JuliaGraphics/FreeTypeAbstraction.jl/issues/67.

File: src/canvas/dotcanvas.jl [hunk 1]
```diff
@@ -9,17 +9,13 @@ DOT_DECODE[4] = ':'
...
-Similar to the `AsciiCanvas`, the `DotCanvas` only uses
-ASCII characters to draw it's content. Naturally,
-it doesn't look quite as nice as the Unicode-based
-ones. However, in some situations it might yield
-better results. Printing plots to a file is one
-of those situations.
+Similar to the `AsciiCanvas`, the `DotCanvas` only uses ASCII characters to draw it's content.
+Naturally, it doesn't look quite as nice as the Unicode-based ones.
+However, in some situations it might yield better results.
+Printing plots to a file is one of those situations.
```
Comment (waldyrious) [inline src/canvas/dotcanvas.jl:15]: Just dropping a note to commend you for using semantic line breaks instead of a fixed width :)

Comment (t-bltg) [reply src/canvas/dotcanvas.jl:15]: Thanks, I didn't know that it was called that way, it just felt more readable ;)

Comment (waldyrious) [reply src/canvas/dotcanvas.jl:15]: It's actually a [best practice](https://rhodesmill.org/brandon/2012/one-sentence-per-line/) that goes back all the way to the early Unix days :)

Comment (t-bltg) [reply src/canvas/dotcanvas.jl:15]: 1974 ! Wasn't even born, dully noted :+1:

**Pull request merged**